app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="org.mifos.mobilebanking">
+    package="org.mifos.mobilebanking">
```

Comment:
Please do not change the spacing

Reply:
@luckyman20 I thought Travis checks would find that out but it seems Travis check does not work for XML layouts. any more spaces which I have left?